libs/image/website/src/lib/image.context.tsx

```diff
@@ -0,0 +1,22 @@
+import {BuilderImageProviderProps} from '@wepublish/website/builder'
+import {mergeDeepLeft} from 'ramda'
+import {createContext, useContext, useMemo} from 'react'
+
+export const ImageContext = createContext<BuilderImageProviderProps>({})
```

Comment:
Allows now to override props given to the image, this allows one to set square to true and not just set aspect-ratio to 1.